addons/hr/views/hr_employee_views.xml

```diff
@@ -240,7 +240,7 @@
                    <templates>
                        <t t-name="kanban-box">
                        <div class="oe_kanban_global_click o_kanban_record_has_image_fill o_hr_kanban_record">
-                           <field name="image_128" widget="image" class="o_kanban_image_fill_left" options="{'zoom': true, 'zoom_delay': 1000, 'background': true, 'preventClicks': false}"/>
+                           <field name="image_128" widget="image" class="o_kanban_image_fill_left" placeholder="/base/static/img/avatar_grey.png" options="{'zoom': true, 'zoom_delay': 1000, 'background': true, 'preventClicks': false}"/>
```

Comment:
Not using avatar here?

Reply:
Yes, it was requested by the PO that the avatar is not used in kanban view for employees